rest-api-guidelines/guides/api-testing-ci-environment.md

```diff
@@ -31,63 +24,28 @@ The following must be available in the CI environment before testing:
     dredd v2.2.5 (Darwin 16.4.0; x64)
    ```
 
-4. [**Apiary CLI Tool**](https://help.apiary.io/tools/apiary-cli) MUST be installed globally in the CI environment:
-
-   ```text
-    $ gem install apiaryio
-   ```
-
-   ```text
-    $ apiary version
-    0.8.0
-   ```
-
-5. **Apiary API Key** MUST be set in the CI Environment environment variables:
-
-   ```text
-    $ export APIARY_API_KEY=xyz
-   ```
-
-   To obtain an Apiary API key, head to [https://login.apiary.io/tokens](https://login.apiary.io/tokens) \(NOTE: You will need the "ALL" Scope\)
-
 ## Testing an API
 
 ### Test Run Prerequisites
 
 To test an API within the CI environment provisioned as mentioned in the environment prerequisites, you will need the following:
 
-1. The name \(subdomain\) of API project at Apiary
+1. A `swagger.yaml` file with the description of API being tested
```

Comment:
this can also be a remote file (Swaggerhub URL). If the API is public its OAS file is reachable from wherever without any additional headers.

Reply:
Added